index.html

```diff
@@ -4,12 +4,12 @@
         <meta charset="utf-8">
         <meta name="viewport" content="initial-scale=1,width=device-width">
         <title>Wo ist Markt?</title>
-        <link rel="stylesheet" href="lib/normalize/normalize.css"/>
-        <link rel="stylesheet" href="lib/leaflet/leaflet.css"/>
-        <link rel="stylesheet" href="lib/leaflet.awesome-markers/leaflet.awesome-markers.css"/>
-        <link rel="stylesheet" href="lib/leaflet.locatecontrol/L.Control.Locate.min.css"/>
-        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"/>
-        <link href="lib/select2/select2.min.css" rel="stylesheet" />
+        <link rel="stylesheet" href="lib/normalize/normalize.css">
+        <link rel="stylesheet" href="lib/leaflet/leaflet.css">
+        <link rel="stylesheet" href="lib/leaflet.awesome-markers/leaflet.awesome-markers.css">
+        <link rel="stylesheet" href="lib/leaflet.locatecontrol/L.Control.Locate.min.css">
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+        <link href="lib/select2/select2.min.css" rel="stylesheet">
```

Comment:
What do you mean? In html5 it is not needed. I did not use any linter. http://stackoverflow.com/a/7366380